Files/UserControls/MultitaskingControl/IMultitaskingControl.cs

```diff
@@ -10,7 +10,8 @@ public interface IMultitaskingControl
 
         public ObservableCollection<TabItem> Items { get; }
 
-        public List<ITabItem> RecentlyClosedTabs { get; }
+        // RecentlyClosedTabs is now static (shared between all multitasking controls)
+        //public List<ITabItem> RecentlyClosedTabs { get; }
```

Comment:
Ignore this change, it fixes an issue with "Ctrl+Shift+T" but it's unrelated to this PR.